frontends/web/src/routes/account/account.tsx

```diff
@@ -222,6 +239,13 @@ export function Account({
         <Status hidden={!hasCard} type="warning">
           {t('warning.sdcard')}
         </Status>
+        <Dialog open={insured && uncoveredFunds.length !== 0} medium title={t('account.warning')} onClose={() => setUncoveredFunds([])}>
+          <MultilineMarkup tagName="p" markup={t('account.uncoveredFunds', {
+            name: account?.name,
```

Comment:
why the question mark? `account` should be defined, right?

Reply:
Yeah, I was just being paranoid :D